crates/ruff_linter/src/rules/pyupgrade/rules/outdated_version_block.rs

```diff
@@ -44,6 +44,11 @@ use ruff_python_ast::PythonVersion;
 /// ## Options
 /// - `target-version`
 ///
+/// ## Fix safety
+/// This rule's fix is marked as unsafe because it will remove all code,
+/// comments, and annotations within unreachable version blocks—even if
+/// they are never executed at runtime.
```

Comment:
```suggestion
/// comments, and annotations within unreachable version blocks.
```

I think we could drop this last part, or possibly reword it. These should _only_ be removed if they are never executed, not *even* if they are. But that's pretty much the rule description, so we could just delete this part.